### ISSUE TITLE
Local storm.yaml isn't quite right

### ISSUE THREAD

**Issue by ndtreviv** [KasperMadsen/storm-deploy-alternative#44]
Storm.java#142 needs changing to `nimbus.seeds: [ IP HERE ]` for storm 1.0.1 deployments.

**Comment by KasperMadsen** [comment]
This should have been fixed by commit 5b002bc...

I have done a test deployment with 1.0.1 and I managed to get the UI showing both master and supervisors. Please provide your serverside storm.yaml so I can see what problem you are observing.

**Comment by ndtreviv** [comment]
No, this is in the file that is written to the client in the ~/.storm directory

**Comment by KasperMadsen** [comment]
I see, I will fix it now.

**Comment by ndtreviv** [comment]
It's my fault - it should have been part of my change!

**Comment by KasperMadsen** [comment]
No worries, please retry...
